src/components/main/Main.tsx

```diff
@@ -6,13 +6,17 @@ import CityCard from "../city-card/CityCard";
 import AccomodationCard from "../accomodation-card/AccomodationCard";
 import PlaceCard from "../place-card/PlaceCard";
 import AccommodationScreen from "../accomodation-screen/AccommodationScreen";
+import PopularLocations from "../popular-locations/PopularLocations";
+import HomesGuestsLove from "../homes-guests-love/HomesGuestsLove";
 
 const Main: React.FC = () => {
     return (
         <>
             <Navigation/>
             <Header/>
+            <PopularLocations />
             <CityCard/>
+            <HomesGuestsLove />
```

Comment:
Ove dvije komponente su gotovo identične. Možeš li to ostaviti kao jednu komponentu imena primjerice `SectionTop` (ili nešto slično tome), i da ta komponenta prima props koji izgleda npr:
```
{
  title: string,
  arrowLabel: string
}
```
Onda ovdje u `Main.tsx` u liniji 17 staviš: `<SectionTop title="Popular Locations" arrowLabel="View all locations" />`